apps/site/package.json

```diff
@@ -73,11 +76,13 @@
   "devDependencies": {
     "@eslint/compat": "~1.2.7",
     "@next/eslint-plugin-next": "15.2.0",
+    "@opennextjs/cloudflare": "^0.6.4",
     "@testing-library/jest-dom": "~6.6.3",
     "@testing-library/react": "~16.2.0",
     "@testing-library/user-event": "~14.6.1",
     "@types/jest": "29.5.14",
     "@types/semver": "~7.5.8",
+    "chokidar": "^4.0.3",
```

Comment:
Why?

Reply:
`node --watch` didn't seem to work for me (https://github.com/nodejs/nodejs.org/pull/7383#discussion_r2025550156) I'm going to try again, sorry 😓

Reply:
chokidar removed 🙂